org.uqbar.project.wollok.launch/src/org/uqbar/project/wollok/launch/tests/DefaultWollokTestsReporter.xtend

```diff
@@ -14,7 +14,7 @@ import wollok.lib.AssertionException
  */
 class DefaultWollokTestsReporter implements WollokTestsReporter {
 	
-	override reportTestAssertError(WTest test, AssertionException assertionError, int lineNumber, URI resource) {
+	override reportTestAssertError(WTest test, AssertionException assertionError, WollokObject wollokException, int lineNumber, URI resource) {
```

Comment:
Added a wollokException in AssertionException, but then I had to use a common String message as a parameter because WollokObject is not Serializable:

```
	override reportTestAssertError(WTest test, AssertionException assertionException, int lineNumber, URI resource) {
		testsResult.add(WollokResultTestDTO.assertionError(test.name, assertionException.message, assertionException.wollokException?.convertStackTrace, lineNumber, resource?.toString))
	}

	override reportTestError(WTest test, Exception exception, int lineNumber, URI resource) {
		testsResult.add(
			WollokResultTestDTO.error(test.name, exception.convertToString, exception.convertStackTrace, lineNumber,
				resource?.toString))
	}
```

I think it is a middle term solution.